lib/wda-mac.js

```diff
@@ -334,7 +379,7 @@ class WDAMacServer {
   }
 
   async stopSession () {
-    if (!this.isRunning) {
+    if (!this.isRunning && !this.isProxyingRemoteWDAMac) {
```

Comment:
isRunning method must be either renamed or patched to respect the remote proxy feature. According to the new logic the server is considered running even if no process is running, but the remote proxy is alive